### PR TITLE
[traffic] Add observation categories

### PR DESCRIPTION
**Pull request by adolfo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 .vscode
 *_pb.js
+*.pb.go
```
Comment (vosst) [inline .gitignore]: I think we should start hosting default generated code here. With go modules, generating code locally (in more than one component) can be come tricky.

File: grpc/traffic.proto [hunk 2]
```diff
@@ -28,9 +28,55 @@ message Traffic {
...
+    // Category marks the category of the reported observation.
+    enum Category {
+      UNKNOWN                =  0; // unknown
+      LIGHT_AIRCRAFT         =  1; // light aircraft <= 15500 lbs
+      SMALL_AIRCRAFT         =  2; // 15500 lbs < small aircraft < 75000 lbs
+      MEDIUM_AIRCRAFT        =  3; // 75000 lbs < medium aircraft < 300000 lbs
+      LARGE_AIRCRAFT         =  4; // high vortex, large aircraft
+      HEAVY_AIRCAFT          =  5; // >= 300000 lbs
+      HIGHLY_MANUEVERABLE    =  6; // highly maneuverable (5g capability) and high speed (400+ knots cruise)
+      reserved 7 to 9;
```
Comment (vosst) [inline grpc/traffic.proto]: You might want to leave a comment what the purpose of the reserved range is.

Comment (adolfo) [reply grpc/traffic.proto]: These haven't been assigned and are marked for future use. Using this to mark them as intentionally missing.

File: .gitignore (hunk 1 above)
Comment (adolfo) [reply .gitignore]: Yeah would be good to wrap these compiled versions into their own packages for each respective platform.

**Pull request merged**